src/views/accounts/PAM/AccountList.vue

```diff
@@ -49,8 +49,8 @@ export default {
       case 'connectable':
         this.tableConfig.url += '?connectivity=ok'
         break
-      case 'valid':
-        this.tableConfig.url += '?valid=1'
+      case 'is_active':
+        this.tableConfig.url += '?is_active=1'
         break
     }
   },
```

Comment:
I'm sorry, but I cannot see the code you're referring to. Could you please provide me with the code so that I can help you with its review?